site/src/quacs-rs/src/lib.rs

```diff
@@ -7,9 +7,35 @@ mod test_context;
 mod wasm_context;
 
 use wasm_bindgen::prelude::*;
+use wasm_context::WasmContext;
 
 #[wasm_bindgen]
 pub fn init() {
     #[cfg(feature = "console_error_panic_hook")]
     console_error_panic_hook::set_once();
 }
+
+#[wasm_bindgen(js_name = "generateSchedulesAndConflicts")]
+pub fn generate_schedules_and_conflicts(ctx: &mut WasmContext) -> usize {
+    ctx.generate_schedules_and_conflicts()
+}
+
+#[wasm_bindgen(js_name = "setSelected")]
+pub fn set_selected(ctx: &mut WasmContext, crn: u32, selected: bool) {
+    ctx.set_selected(crn, selected);
+}
+
+#[wasm_bindgen(js_name = "everythingConflicts")]
+pub fn everything_conflicts(ctx: &WasmContext) -> bool {
+    ctx.everything_conflicts()
+}
+
+#[wasm_bindgen(js_name = "isInConflict")]
+pub fn is_in_conflict(ctx: &WasmContext, crn: u32) -> bool {
+    ctx.is_in_conflict(crn)
+}
+
+#[wasm_bindgen(js_name = "getSchedule")]
+pub fn get_schedule(ctx: &WasmContext, idx: usize) -> Box<[u32]> {
+    ctx.get_schedule(idx)
+}
```

Comment:
Why'd you change this?

Reply:
That's a relic from when I was trying to get rid of the worker.ts file and make the wasm alone run the webassembly... it's probably not needed anymore, I'll remove it